### PR TITLE
Add topics to config file

### PR DESCRIPTION
**Pull request by PatrickMcSweeny**
I noticed that the progress bars in the 'concepts' section weren't updating and I informed that this was because the exercises don't have topics assigned to them. This isn't a complete fix, but it's a start.

### CODE REVIEW

**Comment by Insti** [comment]
Awesome, thanks.

Unfortunately there seems to be a problem with the config.json see: https://travis-ci.org/exercism/ruby/builds/405892767#L1809

Do you think you could look through the file again and see if you can find the problem?

(It's also possible to run the `configlet` command on your own machine so you can check it there.)

File: config.json [hunk 1]
```diff
@@ -11,87 +11,124 @@
...
     },
     {
       "slug": "gigasecond",
       "uuid": "0fb594a1-193b-4ccf-8de2-eb6a81708b29",
       "core": false,
       "unlocked_by": null,
       "difficulty": 1,
-      "topics": null
+      "topics": [
+        "time",
```
Comment (Insti) [inline config.json]: It might be this comma

**Comment by PatrickMcSweeny** [comment]
I fixed the issue with the trailing comma and added a few more topics in the process.

**Comment by Insti** [comment]
Great! Do you want to keep adding topics or shall we merge this as it is now?

**Comment by PatrickMcSweeny** [comment]
This is all I want to do for right now. I will probably add more topics later as I complete more of the exercises myself.

**Comment by Insti** [comment]
Thanks @PatrickMcSweeny :heart:

**Pull request merged**